src/Modules/OrchardCore.Commerce/Views/ProductPart.cshtml

```diff
@@ -8,12 +11,33 @@
 
     // As long as any items can be bought, "Add to cart" button should remain possible to click.
     var canBeBought = Model.CanBeBought.Any(keyValuePair => keyValuePair.Value);
+
+    var minOrderQuantity = 1;
+    var maxOrderQuantity = 1;
+
+    var inventoryPart = Model.ProductPart.ContentItem.As<InventoryPart>();
+    if (inventoryPart != null)
+    {
+        minOrderQuantity = (int)(inventoryPart.MinimumOrderQuantity.Value > 0 && !inventoryPart.IgnoreInventory.Value
+            ? inventoryPart.MinimumOrderQuantity.Value
+            : 0);
+        maxOrderQuantity = (int)(inventoryPart.MaximumOrderQuantity.Value > 0 && !inventoryPart.IgnoreInventory.Value
+            ? inventoryPart.MaximumOrderQuantity.Value
+            : 1);
```

Comment:
The default value for max shouldn't be 1. For example if I fill out the minimum but not the maximum 
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/4007293/3327cde1-6fda-4352-b766-31e7a3163dd7)
then this sets it to `min="2" max="1"` which is invalid.
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/4007293/9b099947-8ec6-4a09-8232-1c4a9ece5b9b)

It's probably fine to use `int.MaxValue` instead of 1.